popdat/src/make_person.rs

```diff
@@ -1,158 +1,221 @@
-use std::collections::HashSet;
+use std::collections::HashMap;
 
 use rand::seq::SliceRandom;
 use rand::Rng;
 use rand_xorshift::XorShiftRng;
 
+use abstutil::{Parallelism, Timer};
 use map_model::{BuildingID, IntersectionID, Map, PathConstraints, PathRequest};
 use sim::{IndividTrip, PersonSpec, TripEndpoint, TripMode, TripPurpose};
 
 use crate::{Activity, CensusPerson, Config};
 
-pub fn make_person(
-    person: CensusPerson,
+pub fn make_people(
+    people: Vec<CensusPerson>,
     map: &Map,
+    timer: &mut Timer,
     rng: &mut XorShiftRng,
     config: &Config,
-) -> PersonSpec {
-    let schedule = person.generate_schedule(config, rng);
-
-    let mut output = PersonSpec {
-        orig_id: None,
-        origin: TripEndpoint::Bldg(person.home),
-        trips: Vec::new(),
-    };
-
+) -> Vec<PersonSpec> {
     // Only consider two-way intersections, so the agent can return the same way
     // they came.
     // TODO: instead, if it's not a two-way border, we should find an intersection
     // an incoming border "near" the outgoing border, to allow a broader set of
     // realistic options.
     // TODO: prefer larger thoroughfares to better reflect reality.
-    let commuter_borders: Vec<IntersectionID> = map
+    let commuter_borders: Vec<map_model::IntersectionID> = map
```

Comment:
nit: `IntersectionID` is already imported

Reply:
fixed!